Source/WebCore/rendering/TextBoxPainter.cpp

```diff
@@ -329,12 +329,15 @@ void TextBoxPainter<TextBoxPath>::paintForegroundAndDecorations()
 
             bool shouldPaintDraggedContent = !(m_paintInfo.paintBehavior.contains(PaintBehavior::ExcludeSelection));
             if (shouldPaintDraggedContent) {
-                auto markedTextsForDraggedContent = MarkedText::collectForDraggedContent(m_renderer, m_selectableRange);
+                auto markedTextsForDraggedContent = MarkedText::collectForDraggedAndTransparentType(DocumentMarker::Type::DraggedContent, m_renderer, m_selectableRange);
```

Comment:
Nit - I think `collectForDraggedAndTransparentContent` might be a slightly more accurate name.